src/OrbitQt/orbitmainwindow.h

```diff
@@ -140,6 +141,7 @@ class OrbitMainWindow : public QMainWindow {
   void SetupMainWindow();
   void SetupHintFrame();
   void SetupTargetLabel();
+  QStringList FilterCommandLineArgs(std::vector<std::string> command_line_args);
```

Comment:
`[[nodiscard]] QStringList FilterCommandLineArgs(const std::vector<std::string>& command_line_args) const;`

Reply:
Thanks for pointing out this. I modified this method as `void FilterCommandLineArgs(const std::vector<std::string>& command_line_args, QStringList& command_line_flags, std::vector<std::string>& capture_file_paths)`  and put it in main.cpp.

The new method extracts both capture file paths and the command line flags. If multiple capture file paths are provided, we starts multiple Orbit instance to open the provided capture files.